client/my-sites/comments/comment-list/index.jsx

```diff
@@ -288,7 +295,7 @@ export class CommentList extends Component {
 		this.props.successNotice( message, noticeOptions );
 	}
 
-	toggleBulkEdit = () => this.setState( { isBulkEdit: ! this.state.isBulkEdit } );
+	toggleBulkEdit = () => this.setState( ( { isBulkEdit } ) => ( { isBulkEdit: ! isBulkEdit } ) );
```

Comment:
Do you mind updating the whitespace here so we don't misread? How about the following?

```jsx
toggleBulkEdit = () => {
    return this.setState( ( { isBulkEdit } ) => ( { isBulkEdit: ! isBulkEdit } ) );
}
```